client/src/searchPage/filterPanel/filterPanel.scss

```diff
@@ -1,26 +1,24 @@
 .filter-wrapper {
+  max-width: 65rem;
+  width: 100%;
+  margin: 0 auto;
+  position: static;
+  @include breakpoint($wk-target-max-desktop) {
```

Comment:
I guess there should be a new line before *include* as well. Please fix every other document if you don't object

Reply:
Updated.